swift/Sources/Keystore/Watch.swift

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2017-2018 Trust.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+import Foundation
+
+/// An account watch.
+public struct Watch: Codable {
+    /// Coin type.
+    public var coin: CoinType
+
+    /// Account address.
+    public var address: String
+
+    /// Account xpub.
+    public var xpub: String?
+
+    public init(coin: CoinType, address: String, xpub: String?) {
+        self.coin = coin
+        self.address = address
+        self.xpub = xpub
+    }
+
+    enum CodingKeys: CodingKey {
+        case coin
+        case address
+        case xpub
+    }
+
```

Comment:
can we add test case to make sure it saves and it loads?

Reply:
Done